docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -708,13 +708,30 @@ Displays debug information in the upper right corner. This can be useful for deb
 - **Zoom** displays current zoom level of the map.
 - **Center** displays the current **longitude**, **latitude** of the map center.
 
-{{< figure src="/static/img/docs/geomap-panel/geomap-map-controls-debug-9-1-0.png" max-width="1200px" alt="Geomap panel debug" >}}
+{{< figure src="/static/img/docs/geomap-panel/geomap-map-controls-debug-9-1-0.png" max-width="400px" alt="Geomap panel debug" >}}
 
 #### Tooltip
 
+Tooltips are supported for the **Markers**, **Heatmap**, **Photos** (beta) layers.
```

Comment:
We also support it in Dynamic GeoJSON now too!

Reply:
Noted. Right now we haven't even documented that particular map layer, so let's leave it out of this PR and handle that separately.